.circleci/config.yml

```diff
@@ -247,7 +247,7 @@ commands:
           # detect flakyness.
           condition: ${CIRCLE_PULL_REQUEST}
           steps:
-            set-retry-flaky-tests
+            - set-retry-flaky-tests
```

Comment:
Does the old version not work?

Reply:
CircleCI flags this as an incorrect data type error which means it could easily stop working in future releases.

Reply:
Interesting, but they don't show it as a config error.  I wonder why not?   LGTM to this part either way.

Reply:
https://circleci.com/docs/vs-code-extension-overview/#config-helper is in-context help..